src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -541,6 +541,67 @@ public function dumpFile($filename, $content, $mode = 0666)
     }
 
     /**
+     * Returns the "chmod" of the file or directory
+     *
+     * @param string $path A file or directory path.
+     * @param boolean $groups Get file|directory permissions by groups ex: 'u-wxrw--wt'.
+     *                        formats:
+     *                        s => socket;
+     *                        l => symbolic link;
+     *                        - => regular;
+     *                        b => block special;
+     *                        d => directory;
+     *                        c => character special;
+     *                        p => FIFO pipe;
+     *                        u => unknown.
+     *
+     * @return string Permissions of the file|directory or throw an exception on failure.
+     */
+    public function permissions($path, $groups = false)
+    {
```

Comment:
Personally I would prefer to have a constant defining the flags of groups.  

What about calling it `getPremissions()` instead? That would be a lot easier to use as you expect to get something thus start guessing the method name like that.